docs/book/deploying-zenml/zenml-cloud/zenml-cloud.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Your one-stop MLOps control plane.
+---
+
+# ☁ ZenML Cloud
+
+ZenML Cloud is a Software-as-a-Service (SaaS) platform that extends the capabilities of the open-source ZenML product. It provides users with a centralized control plane to effortlessly launch and manage their ZenML server instances. The core foundation of ZenML Cloud remains the powerful open-source offering, but with ZenML Cloud, users gain access to a host of additional features designed to streamline the machine learning workflow.
```

Comment:
This paragraph talks about 'users', but later in the page it talks about 'you' / 'your ml workflows' etc. I find 'you / your' nicer here, but in any case maybe consider standardising the style / how this is phrased?

Reply:
Good idea! Changed it accordingly.